packages/orbit-components/src/SegmentedSwitch/index.tsx

```diff
@@ -36,16 +36,18 @@ const SegmentedSwitch = ({
   }, [showTooltip, hasTooltip, setTooltipShown]);
 
   return (
-    <label
+    <div
       data-test={dataTest}
       ref={labelRef}
       className={cx(
         "gap-50 relative flex w-full flex-col",
         spaceAfter && spaceAfterClasses[spaceAfter],
-        "[&_.orbit-switch-segment-label:nth-child(odd)_.orbit-switch-segment-text]:rounded-s-[5px]",
-        "[&_.orbit-switch-segment-label:nth-child(even)_.orbit-switch-segment-text]:rounded-e-[5px]",
+        "[&_.orbit-switch-segment-label:first-child_.orbit-switch-segment-text]:rounded-s-[5px]",
+        "[&_.orbit-switch-segment-label:last-child_.orbit-switch-segment-text]:rounded-e-[5px]",
       )}
       style={{ maxWidth }}
+      role="group"
+      aria-label={label}
```

Comment:
Why don't we also open `ariaLabel` prop and set `aria-label` to either provided `ariaLabel` or to `label` when not provided? I can imagine scenario when one doesn't want to provide visible label, but want to be able to provide aria-label 🤔

Reply:
I thought of that, but that should not be norm. Visible labels should be standard. But I guess it doesn't harm to add